pkg/oc/cli/cmd/set/deployment_strategy.go

```diff
@@ -0,0 +1,481 @@
+package set
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"k8s.io/apimachinery/pkg/api/meta"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/kubernetes/pkg/apis/apps"
+	"k8s.io/kubernetes/pkg/apis/extensions"
+	"k8s.io/kubernetes/pkg/kubectl/cmd/templates"
+	kcmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+	"k8s.io/kubernetes/pkg/kubectl/resource"
+
+	deployapi "github.com/openshift/origin/pkg/apps/apis/apps"
```

Comment:
@mfojtik you should use /v1 that will also make sure that maxUnavailable and friends are pointers and calculate correct patches

Reply:
I don't think the CLI factory, info and builder will handle that switch well and I'm not going to rewrite them from scratch. @deads2k @liggitt not sure if we are going to switch the CLI to external types anytime soon.